<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegoUSCaliWidget</class>
 <widget class="QWidget" name="LegoUSCaliWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>28</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="USAcqObjectLabel">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>US acquisition:               </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="acqPushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="initialization">
       <property name="text">
        <string>Initialization xfm:           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="initxfmLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="initxfmPushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="outputlabel">
       <property name="text">
        <string>Output folder:                </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputPushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="samplerate">
       <property name="text">
        <string>Sampling rate:                    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="samplerateComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="param_1">
       <property name="text">
        <string>Simplex:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SimplexComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="param_2">
       <property name="text">
        <string>Step size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="StepComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
